HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -4481,7 +4481,7 @@ This prevents the mouse cursor appearing under certain conditions, even if the p
     }
 }
 
-Patch "Always Play 'One Way Dream' During Credits" in "Audio" by "M&M"
+Patch "Always Play 'One Way Dream' During Credits" in "Audio" by "M&M" does "Plays 'One Way Dream' in place of 'Vandalize' during the credits in Normal and Easy modes."
```

Comment:
- Change "in place of" to "instead of".
- Change "in Normal and Easy modes" to "on Easy or Normal difficulty".